apps/mark-scan/backend/src/util/auth.ts

```diff
@@ -1,3 +1,4 @@
+/* istanbul ignore next */
```

Comment:
nit: slight preference for ignoring at the `jest.config` level so we have an inventory of what files we're ignoring

Reply:
[ae0bad8](https://github.com/votingworks/vxsuite/pull/4628/commits/ae0bad8230459bc59a5f5ab2338f12a86326b9bc)